webapp-django/crashstats/crashstats/jinja2/crashstats/report_index.html

```diff
@@ -644,6 +651,64 @@
                     </ul>
                 </div>
                 <!-- /correlation -->
+
+                {% if request.user.has_perm('crashstats.reprocess_crashes') %}
+                <div id="reprocess" class="ui-tabs-hide" data-crash-id="{{ report.uuid }}">
```

Comment:
I've gotten so used to `data-` attributes because they're so convenient with jQuery because they support types. 
Also, a `<input>` tag would add to the DOM which is odd for something that will never be displayed. 

Also, the point of putting it in the form would only be to be able to use `$(form).serialize()` or something. I'm not convinced it's worth it.